apps/omg_child_chain_rpc/test/omg_child_chain_rpc/web/plugs/health_test.exs

```diff
@@ -93,7 +101,10 @@ defmodule OMG.ChildChainRPC.Plugs.HealthTest do
   defp pull_client_alarm(0, _, _), do: :cant_match
 
   defp pull_client_alarm(n, match, fnn) do
```

Comment:
the name of this function is also a bit confusing
match now also does not quite mean the same as the match in the elixir convention

Reply:
what's confusing about the name of the function? please, stick reviewing to what the PR is fixing. this went through a review already since it's in master.